src/app/lib/utilities/srcSet/index.js

```diff
@@ -1,6 +1,10 @@
 import buildIChefUrl from '#lib/utilities/ichefURL';
 
 const DEFAULT_RESOLUTIONS = [240, 320, 480, 624, 800];
+const PIPS_RESOLUTIONS = [
```

Comment:
I don't think we need so many image sizes for our app. this will create a lot of markup. can we just have the same resolutions that we have for DEFAULT_RESOLUTIONS? These image sizes are all we really need for responsive images to be performant in simorgh

Reply:
Sure, I just reverted the file back to use only DEFAULT_RESOLUTIONS